templates/pcw.ini

```diff
@@ -17,26 +17,39 @@ smtp-port = 25
 to = <your_email@suse.com>
 from = pcw@publiccloud.qa.suse.de
 
+
+[cluster.notify]
+# List of people who will be notfied about found EKS clusters
+to = <your_email@suse.com>
+
 [notify.namespace.qac]
 # list of recipients which will be notified in case of issues in certain namespace
 to = <your_email@suse.com>
 
 [cleanup]
-# Specify how many images per flavor get kept
-max-images-per-flavor = 2
-# Max age of an image file
-max-images-age-hours = 24
+# Max age of data storage resources ( used in EC2 only )
+ec2-max-age-days = 1
+# Max age of data storage resources ( used in Azure and GCE )
+max-age-hours = 1
 # Specify with which namespace, we will do the cleanup.
 # if not specifed default/namespaces list will be taken instead
 namespaces = qac
 
 # Values specified under "cleanup.namespace.{namespace}" have precedence over same values in [cleanup] for this certain namespace
 [cleanup.namespace.qac]
-# EC2 snapshots younger than this amount of days will be ignored
-ec2-max-snapshot-age-days = 2
-# EC2 volumes younger than this amount of days will be ignored
-ec2-max-volumes-age-days = 2
+# Max age of an image file ( used in EC2 only )
+ec2-max-age-days = 2
+# Azure resource group which PCW will scan for data storage resources needs to be deleted
 azure-storage-resourcegroup = openqa-upload
+# AccountName used for creation of BlobServiceClient
 azure-storage-account-name = openqa
 # When set to true EC2 VPC cleanup will be enabled
 vpc_cleanup = true
+
+[updaterun]
+# if openqa_ttl tag is not defined this TTL will be set to the instance
```

Comment:
Please define, what unit this ttl is in (hours? days? bananas?)

Reply:
done